nimbus.fml.yaml

```diff
@@ -215,13 +215,20 @@ types:
       description: The identifiers for the sections of the MR 2022.
       variants:
         feature-ui:
-          description: Indicates if the user interfaces should be visible.
+          description: An integer either 0 or 1 indicating if the UI for cookie banner handling should be visible,
+          0 to hide the UI and 1 to show the UI. The actual UI is composed by cookie banner section 
+          in the settings page, the toolbar section and the re-engagement dialog.
```

Comment:
I think the failure is complaining because of missing indents here.

Reply:
Thanks!